KPI/L.adoc

```diff
@@ -1,73 +1,157 @@
+                    <gmd:electronicMailAddress>
+                      <gco:CharacterString>gisc_support@meteo.fr</gco:CharacterString>
+                    </gmd:electronicMailAddress>
+                  </gmd:CI_Address>
+                </gmd:address>
+                <gmd:onlineResource>
+                  <gmd:CI_OnlineResource>
+                    <gmd:linkage>
+                      <gmd:URL>http://www.meteo.fr</gmd:URL>
+                    </gmd:linkage>
+                  </gmd:CI_OnlineResource>
+                </gmd:onlineResource>
+              </gmd:CI_Contact>
+            </gmd:contactInfo>
+            <gmd:role>
+              <gmd:CI_RoleCode codeListValue="pointOfContact" codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/Codelist/gmxCodelists.xml#CI_RoleCode">pointOfContact</gmd:CI_RoleCode>
+            </gmd:role>
+          </gmd:CI_ResponsibleParty>
+        </gmd:distributorContact>
+      </gmd:MD_Distributor>
+    </gmd:distributor>
+    <gmd:transferOptions>
+      <gmd:MD_DigitalTransferOptions>
+        <gmd:onLine>
+          <gmd:CI_OnlineResource>
+            <gmd:linkage>
+              <gmd:URL>http://wispi.meteo.fr/openwis-user-portal/srv/en/main.home?urn=urn:x-wmo:md:int.wmo.wis::ISMN10LFPW</gmd:URL>
+            </gmd:linkage>
+            <gmd:protocol>
+              <gco:CharacterString>WWW:LINK-1.0-http--link</gco:CharacterString>
+            </gmd:protocol>
+            <gmd:name>
+              <gco:CharacterString>Permanent link</gco:CharacterString>
+            </gmd:name>
+            <gmd:description>
+              <gco:CharacterString>GISC Toulouse</gco:CharacterString>
+            </gmd:description>
+          </gmd:CI_OnlineResource>
+        </gmd:onLine>
+      </gmd:MD_DigitalTransferOptions>
+    </gmd:transferOptions>
+  </gmd:MD_Distribution>
 </gmd:distributionInfo>
-....
+```
+
+=== What is being measured
+
+The presence of distribution information and supporting elements.
+
+* format (`//gmd:MD_Metadata/gmd:distributionInfo/*/gmd:distributionFormat`).
+  That can rely on existing WMO supported formats
+* 1..n `gmd:transferOptions` (`//gmd:MD_Metadata/gmd:distributionInfo/*/gmd:transferOptions//gmd:onLine`),
+  including a `gmd:linkage` with a URL to access the service providing the data
+* distributor contact information (organization, email)
+ 
+=== Rationale for measurement
+
+Distribution information provides the necessary information for accessing the
+data, supported formats, and contact information for the data distributor.
+
+=== Rules for implementation
+
+By detecting the presence of the distribution format and transferOptions
+(multiple options are possible). See page 35 of the WMO Core Profile Metadata
+Guidance.
 
-==== E.2 What is being measured?
+.Distribution information implementation rules
+[cols="1a,2"]
+|===
+|Rule |Score
 
-The presence of such element and its subelements:
+|format is defined
```

Comment:
We mentioned to make more explicit gmd:MD_Format is present +1 (instead of format)  (and then by definition of the schema that would imply that both name and version are also present )

Reply:
Thanks @martaguti .  Fixed in cec5995.  Let me know if this works.